src/screens/ListScreen.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { array, func, shape } from 'prop-types';
+import { ScrollView, StyleSheet, View, ViewPropTypes } from 'react-native';
+
+import SearchCard from '../components/SearchCard';
+
+const styles = StyleSheet.create({
+  list: {
+    width: '100%',
+    height: '100%',
+    backgroundColor: '#eee',
+    paddingTop: 140,
+  },
+});
+
+export default function ListScreen({ navigation, style, data }) {
+  return (
+    <ScrollView style={[styles.list, style]}>
+      <View style={{ marginTop: 48, marginBottom: 248 }}>
```

Comment:
this may become an issue if we have phones with different screen sizes, not 100% familiar with css padding however